versioned_docs/version-2.6/pages-for-subheaders/authentication-config.md

```diff
@@ -109,3 +109,23 @@ If you need to reconfigure or disable then re-enable a provider that had been pr
 is logged in to Rancher as an external user, not the local admin.
 
 :::
+
+## Disabling an auth provider
+
+When you disable an auth provider, Rancher deletes all resources associated with it, including secrets,
+global role bindings, cluster and role template bindings, and users who had never been local users in Rancher.
+
+As this operation may lead to a loss of many resources, you may want to add a saferuard on the provider.
+To ensure this cleanup process doesn't run when the auth provider is disabled, add a special annoation to the corresponding auth config.
```

Comment:
```suggestion
Since disabling the auth provider may lead to a loss of many resources, you may want to add a saferuard. To ensure that the cleanup process doesn't run when the auth provider is disabled, add a special annotation to the corresponding auth config.
```

Saferuard == safeguard?